### PR TITLE
Age sex

### PR DESCRIPTION
**Pull request by kya2**
- Added date filter (2019-01-01 and later)
- Added sous coord, top zone, and month figures
- Added tables

### CODE REVIEW

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd [hunk 1]
```diff
@@ -165,13 +165,16 @@ We retain only entries:
...
+* from 2019-01-01 to present
 
 ```{r filter}
 
 x <- x %>%
   filter(epicasedef %in% c("confirmed", "probable"),
          sex %in% c("male", "female"),
-         age_class != "unknown")
+         age_class != "unknown",
+         month_report >= as.Date("2019-01-01"))
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: This works but safer to use the date of reporting (I think `date_report`) as `month_report` is derived from it.

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd [hunk 2]
```diff
@@ -397,14 +419,17 @@ have been generated and are in the same folder as this document):
 Explain what the exported files are using the following model:
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: Remove this line as it was part of the template.

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd [hunk 3]
```diff
@@ -397,14 +419,17 @@ have been generated and are in the same folder as this document):
 Explain what the exported files are using the following model:
 
 
-* `produced_rds/tab_overall.rds`: tab_overall contains the overall distributions of cases by sex.
+* `produced_rds/tab_overall.rds`: tab_overall contains the overall distributions of cases by age group and sex.
```
Comment (thibautjombart) [inline linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: Remove the whole rds export section as probably not useful for further analyses in R.

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd (hunk 1 above)
Comment (kya2) [reply linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: updated code accordingly

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd (hunk 2 above)
Comment (kya2) [reply linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: Deleted " (only works if the files above have been generated and are in the same folder as this document)"

File: linelist_investigations/report_sources/age_sex_2019-10-18.Rmd (hunk 3 above)
Comment (kya2) [reply linelist_investigations/report_sources/age_sex_2019-10-18.Rmd]: Edited accordingly

**Comment by thibautjombart** [comment]
Thanks @kya2 ! I have made minor changes to the graph, see comparison on :
https://github.com/reconhub/report_factories_templates/pull/3/commits/88f4015005f98d1aa9f1811117870823d47480b1

Can you test that version on your computer? You just need to pull the current branch and see if the report compiles fine.

**Comment by thibautjombart** [comment]
Sounds cool, merging! Thanks and congrats on the first successful PR! :)

**Pull request merged**